### DISCUSSION TITLE
sftp upload always corrupts .docx word file

### DISCUSSION THREAD

**Discussion by andrej-taranenko** [iterate-ch/cyberduck#15881] [Q&A]
We use an sftp server at work. Whenever I upload a .docx word file it gets corrupted. The content of the uploaded and the original file is different with no errors during transfers. I checked this with the upload and then download compared to the original.

If I put word files into a .zip file and upload it, everything works ok, I can download, unzip and see the uncorrupted word file.

Are .docx files perhaps transfered in text mode instead of binary? Can this be changed in the program, I can't find any such settings.

I am using Cyberduck 8.7.2. on MAC OS Sonoma 14.3.1 (23D60).

**Comment by itswzyss** [comment]
I figured I'd just try to repro just to add context -- I can say that this doesn't seem to affect Windows on the latest version (8.8.1). Not helpful for your case specifically, but at least this narrows it down to the Mac side.
